### PR TITLE
Merge of physical lifestyle to magical lifestyles branch

### PR DESCRIPTION
**Pull request by MemeWizard**
### Developer changelog:
- Changed a few arts
- Deleted `auto_selection_weight` and associated triggers because it should be reworked later, on branch that fix `auto_selection_weight` for both magical lifestyles and physical lifestyles.

## Tests:
- [x] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [x] The mod takes less than 5.5 GB in the Task Manager (Windows)

### CODE REVIEW

**Review by robmart — APPROVED** [review]
LGTM

File: common/focuses/wc_physical_lifestyle_focuses.txt [hunk 1]
```diff
@@ -2,131 +2,43 @@
 	lifestyle = physical_lifestyle
```
Comment (robmart) [inline common/focuses/wc_physical_lifestyle_focuses.txt:2]: Errors:
```[13:35:07][pdx_persistent_reader.cpp:229]: Error: "Unknown trigger type: wc_can_start_new_lifestyle_tree_foreign_affairs_trigger, near line: 2
Unknown trigger type: wc_can_start_new_lifestyle_tree_majesty_trigger, near line: 3
Unknown trigger type: wc_can_start_new_lifestyle_tree_family_trigger, near line: 4
Unknown trigger type: wc_can_start_new_lifestyle_tree_strategy_trigger, near line: 5
Unknown trigger type: wc_can_start_new_lifestyle_tree_authority_trigger, near line: 6
Unknown trigger type: wc_can_start_new_lifestyle_tree_chivalry_trigger, near line: 7
Unknown trigger type: wc_can_start_new_lifestyle_tree_wealth_trigger, near line: 8
Unknown trigger type: wc_can_start_new_lifestyle_tree_domain_trigger, near line: 9
Unknown trigger type: wc_can_start_new_lifestyle_tree_duty_trigger, near line: 10
Unknown trigger type: wc_can_start_new_lifestyle_tree_skulduggery_trigger, near line: 11
Unknown trigger type: wc_can_start_new_lifestyle_tree_temptation_trigger, near line: 12
Unknown trigger type: wc_can_start_new_lifestyle_tree_intimidation_trigger, near line: 13
Unknown trigger type: wc_can_start_new_lifestyle_tree_medicine_trigger, near line: 14
Unknown trigger type: wc_can_start_new_lifestyle_tree_scholarship_trigger, near line: 15
Unknown trigger type: wc_can_start_new_lifestyle_tree_theology_trigger, near line: 16
" in file: " file: common/focuses/00_lifestyle_focuses.txt line: 33; can_start_new_lifestyle_tree_trigger" near line: 42
[13:35:07][pdx_persistent_reader.cpp:229]: Error: "Unknown trigger type: wc_can_start_new_lifestyle_tree_foreign_affairs_trigger, near line: 34
" in file: "common/focuses/00_lifestyle_focuses.txt" near line: 35
[13:35:07][pdx_persistent_reader.cpp:229]: Error: "Unknown trigger type: wc_lifestyle_tree_foreign_affairs_finished_trigger, near line: 39
" in file: "common/focuses/00_lifestyle_focuses.txt" near line: 39
[13:35:07][pdx_persistent_reader.cpp:229]: Error: "Unknown trigger type: wc_can_start_new_lifestyle_tree_foreign_affairs_trigger, near line: 2
```

I think that's all of them, but they were repeating over and over in the logs.

**Review by robmart — CHANGES_REQUESTED** [review]

**Review by robmart — APPROVED** [review]
image

No errors related to physical stuff, re-approving

File: common/focuses/wc_physical_lifestyle_focuses.txt (hunk 1 above)
Comment (MemeWizard) [reply common/focuses/wc_physical_lifestyle_focuses.txt:2]: Errors related to missing triggers should be fixed when #633 is merged

**Review by zumbak04 — APPROVED** [review]
LGTM! Though the branch has outdated triggers like:
```wc_can_start_new_lifestyle_tree_foreign_affairs_trigger```

**Comment by MemeWizard** [comment]
> Though the branch has outdated triggers like

Yeah, it's true, but I'm afraid that the problem lies on the `magic_trees` branch itself. It should be fixed when #633 will be merged.

@robmart @zumbak04 thanks for reviews <3

**Pull request merged**